opts/env_test.go

```diff
@@ -4,27 +4,38 @@ import (
 	"fmt"
 	"os"
 	"runtime"
+	"strings"
 	"testing"
 )
 
+func TestValidateEnvError(t *testing.T) {
+	expected := "invalid environment variable"
+	invalids := []string{"=b", "="}
+	for _, value := range invalids {
+		_, err := ValidateEnv(value)
+		if err == nil {
+			t.Fatal("expected ValidateEnv to return error")
+		}
+		if !strings.Contains(err.Error(), expected) {
+			t.Errorf("expected to see '%s' in '%s'", expected, err)
+		}
+	}
+}
+
 func TestValidateEnv(t *testing.T) {
 	valids := map[string]string{
-		"a":                   "a",
-		"something":           "something",
-		"_=a":                 "_=a",
-		"env1=value1":         "env1=value1",
-		"_env1=value1":        "_env1=value1",
-		"env2=value2=value3":  "env2=value2=value3",
-		"env3=abc!qwe":        "env3=abc!qwe",
-		"env_4=value 4":       "env_4=value 4",
-		"PATH":                fmt.Sprintf("PATH=%v", os.Getenv("PATH")),
-		"PATH=something":      "PATH=something",
-		"asd!qwe":             "asd!qwe",
-		"1asd":                "1asd",
-		"123":                 "123",
-		"some space":          "some space",
-		"  some space before": "  some space before",
-		"some space after  ":  "some space after  ",
+		"a":                  "a=",
+		"asd!qwe":            "asd!qwe=",
```

Comment:
Nope; we don't validate env var names; IIRC there's no official standard for it, and we did validation in the past, but broke people that used systems that don't comply with those rules.

We leave the validation to the container itself to tell if it's valid or not